bluebrain/repo-patches/packages/neuron/package.py

```diff
@@ -183,6 +185,10 @@ def cmake_enable_option(spec_requiremement):
             compilation_flags += ['-g', '-O0']
             # Remove default flags (RelWithDebInfo etc.)
             args.append("-DCMAKE_BUILD_TYPE=Custom")
```

Comment:
I'm wondering if this variant is still needed since we can just add `build_type=Debug` and the build flags will be propagated for both CoreNEURON and NEURON

Reply:
For simplicity, I think it would be best to rely on `build_type` and not have a custom variant

Reply:
I agree